frontend/dialob-composer-material/src/components/PageTabs.tsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { Menu, MenuItem, Button, IconButton, Box, TextField, Table, TableRow, TableBody, TableContainer, CircularProgress, Typography } from '@mui/material';
+import { Add, Close, ContentCopy, Key, Menu as MenuIcon, Tune, Visibility } from '@mui/icons-material';
+import { DialobItem, DialobItems, useEditor } from "../dialob";
+import { FormattedMessage, useIntl } from "react-intl";
+
+
+const MAX_PAGE_NAME_LENGTH = 40;
+
+const getPageTabTitle = (item: DialobItem, language: string): string => {
+  const rawLabel = item.label ? item.label[language] : null;
+  if (!rawLabel) {
+    return item.id;
+  } else {
+    return rawLabel.length > MAX_PAGE_NAME_LENGTH
+      ? rawLabel.substring(0, MAX_PAGE_NAME_LENGTH) + '…'
+      : rawLabel;
+  }
+};
+
+const PageHeader: React.FC<{ item?: DialobItem }> = ({ item }) => {
+  const intl = useIntl();
+  if (!item) {
+    return null;
+  }
+  return (
+    <TableContainer sx={{ borderRadius: 2, borderTopLeftRadius: 0, border: 1, borderColor: 'divider' }}>
```

Comment:
Please do not replicate the look of old composer. Let's try to remain as pure material as possible. Idea is to replicate functionality. buttons / menus etc. do not have to _look_ like old composer. Things can be moved around if needed. The idea is that composer would eventually fit into DigiExpress management UI, so it should remain themable as such.